pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -352,7 +352,9 @@ def test_validate_inplace_raises(self, value):
             cat.add_categories(new_categories=["D", "E", "F"], inplace=value)
 
         with pytest.raises(ValueError, match=msg):
-            cat.remove_categories(removals=["D", "E", "F"], inplace=value)
+            with tm.assert_produces_warning(FutureWarning):
+                # issue #37643 inplace kwarg deprecated
```

Comment:
nit: I'd drop this comment

Reply:
I was asked to add this comment in my previous pull request #37918, which is related to #37643, so I did the same here

Reply:
yah i find these helpful